docker-entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+set -u # unset variables throw error
+set -o pipefail # pipes fail when partial command fails
+
+bundle update --bundler
+
+# Note: make sure env $DATABASE_* items are set!
+
+# Attempt connect to db and if
+# it fails, then assume db does not exist and
+# db creation should be executed:
+if PGPASSWORD=$DATABASE_PASSWORD psql -lqt -U$DATABASE_USER -h$DATABASE_HOST | cut -d \| -f 1 | grep -q civic; then
+    echo "civic database already exists. Skipping database initialisation."
+else
+    echo "Creating civic database"
+    rake db:create
+
+    echo "Migrating civic database"
+    # migrate:
+    rake db:migrate
```

Comment:
Since you're initializing the database from scratch here, and not running migrations in the case of an update anyways, you could just run `rake db:structure:load` here which will load in the schema in its entirety at once rather than stepping through each migration.

Reply:
@acoffman I've started using the command here https://github.com/griffithlab/civic-server/pull/704/commits/63d762cb3b1d5ce1977f9cbbde4d62759b5a5907